vars/pipelineVegavisorProtocolUpgradeNetwork.groovy

```diff
@@ -38,6 +38,7 @@ void call() {
 
     def versionTag = 'UNKNOWN'
     def protocolUpgradeBlock = -1
+    RELEASE_VERSION = null
```

Comment:
We could make it a string

Reply:
groovy is duck-typed it doesn't matter. For some edge cases it's better to operate on nulls :)